website/code_issues.qmd

```diff
@@ -80,8 +80,55 @@ In your examples, vignettes, demos and tests setting a seed is not only allowed
 
 -----------------------------------------------------------
 
-# Issue 3
+# Using print()/cat()
 
 ## Problem
 
+You are using functions, like print() or cat(), to print unsuppressable output to the console.
+
 ## Solution
+
+Change print()/cat() to message(), warning(), stop(), or wrap them in if(verbose){}.
+
+### Details
+Information messages, like loop counts or status updates, can clutter the console. While some users prefer this display, others appreciate less information on their console. The use of printing functions for console output is not forbidden on CRAN. However, this output must be suppressable by users.
+
+:::{.callout-note}
+
+Printing in special functions like print, summary, interactive functions or methods for generic functions is accepted. 
+
+:::
+
+To allow users to suppress the console output CRAN recommends two different ways: 
+
+  * exchanging cat()/print() with other generics
+    - [message()](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/message): for information messages and status updates 
+    - [warning()](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/warning): for warning, will print a "Warning: " before the output
+    - [stop()](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/stop): for error messages, will print an "Error: " before the output and halt the execution
```

Comment:
CRAN still doesn't provide html manual pages of packages (+base) but it might be better to directly link to those in the r-project.org url: https://search.r-project.org/R/refmans/
Or from Martin Mäechler server: https://stat.ethz.ch/R-manual/R-devel/library/base/html/stop.html

```suggestion
  * exchanging `cat()`/`print()` with other generics
    - [`message()`](https://search.r-project.org/R/refmans/base/html/message.html): for information messages and status updates 
    - [`warning()`](https://search.r-project.org/R/refmans/base/html/warning.html): for warning, will print a "Warning: " before the output
    - [`stop()`](https://search.r-project.org/R/refmans/base/html/stop.html): for error messages, will print an "Error: " before the output and halt the execution
```

Reply:
I got a "Forbidden: You don't have permission to access this resource." when clicking the link https://search.r-project.org/R/refmans/